src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -119,7 +119,7 @@ const TimeEntryForm = props => {
   const [projectsAndTasksOptions, setProjectsAndTasksOptions] = useState([]);
   const [submitting, setSubmitting] = useState(false);
 
-  const canEditTimeEntry = props.hasPermission('editTimelogInfo') || props.hasPermission('editTimeEntry');
+  const canEditTimeEntry = props.hasPermission('editTimelogInfo') || props.hasPermission('editTimeEntry') || props.hasPermission('postTimeEntry');
```

Comment:
Do these need to be separated?

Reply:
Yes, I think it makes sense to separate them.